sklearn/pipeline.py

```diff
@@ -550,7 +567,7 @@ def score_samples(self, X):
 
     @available_if(_final_estimator_has("predict_log_proba"))
     def predict_log_proba(self, X, **predict_log_proba_params):
-        """Apply transforms, and predict_log_proba of the final estimator
+        """Apply transforms, and predict_log_proba of the final estimator.
```

Comment:
```suggestion
        """Apply transforms, and `predict_log_proba` of the final estimator.
```

Reply:
Should we do something similar with the transform method, for example "`transform`s"?

Reply:
since this is used as a verb, in this case, this is fine.

Reply:
"transforms" is precisely not used as verb here :)

Personally I am fine with either options.

Reply:
oh right :) So to be consistent with myself I would say that I prefer `transform`s :)